qa/rolling-upgrade/build.gradle

```diff
@@ -0,0 +1,121 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
+import org.apache.tools.ant.taskdefs.condition.Os
+
+import java.util.concurrent.Callable
+
+apply from : "$rootDir/qa/build.gradle"
+
+String default_bwc_version = System.getProperty("bwc.version")
+String neural_search_bwc_version = System.getProperty("tests.bwc.version", default_bwc_version)
+boolean isSnapshot = neural_search_bwc_version.contains("-SNAPSHOT")
+String knn_bwc_version_no_qualifier = isSnapshot ? neural_search_bwc_version - "-SNAPSHOT" : neural_search_bwc_version
+String baseName = "neuralSearchBwcCluster-rolling"
+
+List<Provider<RegularFile>> plugins = [provider(new Callable<RegularFile>(){
```

Comment:
Can we move these up to qa/build.gradle so we dont have to repeat code between restart and rolling?

Reply:
ACK